src/AssemblerReadGraph.cpp

```diff
@@ -207,32 +235,32 @@ bool Assembler::createLocalReadGraph(
         const uint32_t distance1 = distance0 + 1;
 
         // Loop over edges of the global read graph involving this vertex.
-        for(const uint64_t i: readGraph.connectivity[orientedReadId0.getValue()]) {
+        for (const uint64_t i: readGraph.connectivity[orientedReadId0.getValue()]) {
             SHASTA_ASSERT(i < readGraph.edges.size());
-            const ReadGraphEdge& globalEdge = readGraph.edges[i];
+            const ReadGraphEdge &globalEdge = readGraph.edges[i];
```

Comment:
For consistency with the Shasta standard style and common C++ usage, please put this back as it was, that is, change to the following (space after the `&`, not before):

```
const ReadGraphEdge& globalEdge
```
There is at least one more place below (when defining `alignment`) where a similar change 
 back to the original code is needed.

Reply:
Yes, my mistake, I actually prefer the `&` to be part of the type as well. It must have been swapped by my IDE